solr/core/src/java/org/apache/solr/handler/api/V2ApiUtils.java

```diff
@@ -85,6 +85,11 @@ public static void squashIntoNamedList(
     squashIntoNamedList(destination, mw, false);
   }
 
+  public static void squashIntoNamedListWithoutHeader(
+      NamedList<Object> destination, JacksonReflectMapWriter mw) {
+    squashIntoNamedList(destination, mw, true);
+  }
+
```

Comment:
Here I have introduced new method named `squashIntoNamedListWithoutHeader` as I need to convert JerseyResponse into NamedList without Header. Currently the  method named `squashIntoNamedList` convert JerseyResponse to NamedList but also includes the responseHeader.